internal/cephfs/volumemounter.go

```diff
@@ -83,7 +71,24 @@ func loadAvailableMounters(conf *util.Config) error {
 		if kvErr != nil {
 			return kvErr
 		}
-
+		quotaSupport := []util.KernelVersion{
+			{
```

Comment:
Why this needs to be changed? Just want to understand the reason

Reply:
gomnd warns about using Magic numbers(direct usage of numbers in the code.) The better approach suggested is to access them via variables for easy configuration.

Reply:
This becomes odd as few flags are dined in above variable and few or not. I think we can disable check fo this struct for better readability and maintainability